app/components/thank-you-container.js

```diff
@@ -1,9 +1,103 @@
 import Ember from 'ember';
 
 const {
-  Component
+  Component,
+  computed,
+  get,
+  isPresent
 } = Ember;
 
+const MAX_VOLUNTEERS = 12;
+
+/**
+  The `thank-you-container` component presents the main content for the
+  `thank-you` donations page. This includes an icon, thank you message and
+  a list of contributors.
+
+  ## Default Usage
+
+  ```handlebars
+  {{thank-you-container project=project}}
+  ```
+
+  @class thank-you-container
+  @module Component
+  @extends Ember.Component
+ */
 export default Component.extend({
-  classNames: ['thank-you-container']
+  classNames: ['thank-you-container'],
+
+  /**
+    A computed alias for the project's organization members
+
+    @property organizationMemberships
+    @type Ember.Array
+   */
+  organizationMemberships: computed.alias('project.organization.organizationMemberships'),
+
+  /**
+    Filters the array of `organizationMemberships` down to members who are not
+    pending acceptance.
+
+    @property nonPendingMemberships
+    @type Ember.Array
+   */
+  nonPendingMemberships: computed('organizationMemberships.[]', 'organizationMemberships.@each.role', function() {
+    let memberships = get(this, 'organizationMemberships');
+
+    if (isPresent(memberships)) {
+      return memberships.filter((membership) => {
+        let isPending = get(membership, 'isPending');
```

Comment:
The `organization` model has a line `pendingMemberships: filterBy('organizationMemberships', 'isPending')`. Maybe we could have an `approvedMemberships` and then you can just compute on the `approvedMemberships`?